docs/mrt/mnist_tutorial.md

```diff
@@ -16,25 +19,36 @@
 
 ```bash
 make -j8 lib
-make python
 ```
+> Note that executing `make -j8 lib` for the first time might encounter an error like
+> ```
+Error copying file (if different) from "/alongdirname/cuda_compile_1_generated_broadcast.cu.o.depend.tmp" to "/alongdirname/cuda_compile_1_generated_broadcast.cu.o.depend".
+CMake Error at /alongdirname/cuda_compile_1_generated_broadcast.cu.o.cmake:246 (message):
+	Error generating
+	/alongdirname/./cuda_compile_1_generated_broadcast.cu.o
+build.make:83: recipe for target '/alongdirname/cuda_compile_1_generated_broadcast.cu.o' failed
+> ```
+> Executing it again will fix the problem.
+
 
 ### Mnist Training
+This section gives an example of training a model for mnist dataset and storing the trained model as `~/mrt_model/mnist_dapp.json` and `~/mrt_model/mnist_dapp.params`. CVM is not necessary during this procedure.
 
 Execute the following command:
 
 ```bash
-python3 tests/python/train_mnist.py
+python3 tests/mrt/train_mnist.py
 ```
-
-Training model is stored in `~/mrt_model`.
+> Pay attention to python dependencies. This python script uses GPU defaultly so `mxnet` should be gpu version corresponding to CUDA version.
```

Comment:
Add the CPU helper documentation like:

set the option flag: `--cpu` for CPU usage.